kdc/connect.c

```diff
@@ -965,7 +965,7 @@ loop(krb5_context context, krb5_kdc_configuration *config,
 
 #ifdef __APPLE__
 static void
-bonjour_kid(krb5_context, krb5_kdc_configuration *config, int islive)
+bonjour_kid(krb5_context context, krb5_kdc_configuration *config, int islive)
```

Comment:
It was but I foolishly squashed it. I'll see if I can resurrect it then commit that directly to master.

Sent from my iPhone

> On 29 Nov 2015, at 10:40, Nico Williams <notifications@github.com> wrote:
> 
> In kdc/connect.c:
> 
> > @@ -965,7 +965,7 @@ loop(krb5_context context, krb5_kdc_configuration *config,
> >  
> >  #ifdef __APPLE__
> >  static void
> > -bonjour_kid(krb5_context, krb5_kdc_configuration *config, int islive)
> > +bonjour_kid(krb5_context context, krb5_kdc_configuration *config, int islive)
> If possible make this a separate commit. If not I'll do it when ready to integrate.
> 
> —
> Reply to this email directly or view it on GitHub.
>